File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/sso/LoginLdapResVo.java
package org.javamaster.invocationlab.admin.model.sso;

import lombok.Data;

@Data
public class LoginLdapResVo {
    private String refreshToken;
    private String email;
    private String account;
    private String realName;
    private String token;
    private String blood;
    private String mobileNo;
    private Integer accountType;
    private String sex;
    private String empType;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/sso/LoginLdapResVo.kt
package org.javamaster.invocationlab.admin.model.sso

import org.javamaster.invocationlab.admin.annos.AllOpen


@AllOpen
class LoginLdapResVo {
    var refreshToken: String? = null
    var email: String? = null
    var account: String? = null
    var realName: String? = null
    var token: String? = null
    var blood: String? = null
    var mobileNo: String? = null
    var accountType: Int? = null
    var sex: String? = null
    var empType: String? = null
}
